src/Containers/ProjectForm/ProjectForm.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ProjectForm, mapDispatchToProps } from './ProjectForm';
+import { addProject } from '../../actions/index';
+import { render, fireEvent } from '@testing-library/react';
```

Comment:
This all looks good! It looks like a very similar flow to testing with enzyme. It also sounds like it has some things in place to test Hooks which will be really nice!